extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md

```diff
@@ -50,8 +52,27 @@ multipled by the node's world transform in the shader to produce the final world
 }
 ```
 
-## Appendix
+## Appendix: Motivation and Purpose
+
+*This section is non-normative.*
+
+### Definitions
+
+The term "instancing" has come to mean different things within different parts of the graphics ecosystem. In authoring tools and data formats, it may refer to reuse of any resource for efficiency, whether that resource represents a GPU draw call or not. We will refer to this technique as *data instancing*, and it is unrelated to the `EXT_mesh_gpu_instancing` extension described by this document. On its own, the glTF 2.0 specification provides flexible ways to use data instancing and avoid duplication of meshes, textures, and other low-level resources.
+
+In graphics APIs, the term more commonly refers to drawing a single mesh many times with a single draw call, and a single shader program. Differing vector or scalar parameters (color, position) may be used as inputs to the shader program at each instance.  We will refer to this technique as *GPU instancing*, and it is the focus of the `EXT_mesh_gpu_instancing` extension.
+
+### Motivation
+
+GPU instancing is a common optimization technique, and may be enabled by client implementations with or without this extension — the technique is compatible with, and independent of, the glTF 2.0 specification. What glTF 2.0 does not provide, as of this writing, is an efficient means to transmit very large multiples of mesh data intended for instancing, in a format that indicates that the data has been processed for this purpose. That additional structure is useful for more advanced effects such as fur, grass, dense trees, and buildings.
+
+One tradeoff that any engine must make when considering instancing is in regard to frustum culling. Instances are by definition drawn as a batch, and may (in the worst case) perform worse as a result than if they had been drawn individually, where only a few instances are actually within the visible area. For this reason, the best use of instancing is often an application-specific optimization — for a simple viewer, as single large batch may be appropriate. For a large game world, instances should be grouped into colocated cells (each represented as a node extended with `EXT_mesh_gpu_instancing`) and culled by node bounds. This extension aims to provide the flexibility to optimize data for these different use cases, without prescribing a single solution.
+
+As a result, instancing should be considered a last-mile optimization, similar to compression. The technique is useful to improve runtime performance, but may reduce portability across tools and lose some information (like node hierarchies) from the original asset. For archival purposes, the *data instancing* techniques enabled by the core glTF 2.0 specification should be used instead.
```

Comment:
Great writeup. I'd recommend dropping this sentence `As a result, instancing should be considered a last-mile optimization, similar to compression.`, that does not seem like a useful hint, as it can be misunderstood to say "don't worry about these, they'll only give you a last small x% speedup", when the difference is instead quite dramatic in most cases.

Reply:
That makes sense, thanks!